plugin-volume/volumepopup.cpp

```diff
@@ -41,6 +40,13 @@
 #include "audioengine.h"
 #include <QDebug>
 
+void Slider::wheelEvent(QWheelEvent *event)
+{
+    int delta = event->angleDelta().y() / QWheelEvent::DefaultDeltasPerStep;
+    setValue(value() + delta * singleStep());
+    QTimer::singleShot(0, this, [this] {QToolTip::showText(QCursor::pos(), toolTip());});
```

Comment:
Showing the tooltip is now here and also in `VolumePopup::handleSliderValueChanged()`, I believe it is needed only on one place.